From 2d453cd1bf66962a47e1a06894925c4a9de670c6 Mon Sep 17 00:00:00 2001
From: Ookiineko <chiisaineko@protonmail.com>
Date: Thu, 18 Aug 2022 17:15:20 +0800
Subject: [PATCH 10/17] CMakeLists: set OPENSSL_NO_ASM

- assembly code generated by perlasm does not compile
  well under Cygwin (on my machine, cygwin64), so lets
  disable it for now ig

Signed-off-by: Ookiineko <chiisaineko@protonmail.com>
---
 CMakeLists.txt | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 2447837..be89473 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -6,6 +6,8 @@ endif()
 # Defer enabling C and CXX languages.
 project(BoringSSL NONE)
 
+set(OPENSSL_NO_ASM "1")
+
 if(WIN32)
   # On Windows, prefer cl over gcc if both are available. By default most of
   # the CMake generators prefer gcc, even on Windows.
-- 
2.37.2

